app/views/comfy/admin/blog/posts/_form.html.haml

```diff
@@ -2,9 +2,24 @@
 
 = form.text_field :title, data: {slugify: @post.new_record?}
 = form.text_field :slug, data: {slug: true}
-
+= form.select :file_id, |
```

Comment:
Specifically for images, this could be more of a gallery, but this seemed like a reasonable compromise and especially since `Comfy::Cms::File` isn't strictly for images.